docs/reference/child_admin.rst

```diff
@@ -7,6 +7,10 @@ where the videos will automatically be filtered by post.
 
 To do this, you first need to call the ``addChild`` method in your ``PlaylistAdmin`` service configuration:
 
+To do this, you first need to call the ``addChild`` method in your ``PlaylistAdmin`` service configuration with two arguments,
```

Comment:
You need to add the second part of the sentence to the line above, it is duplicated